### ISSUE TITLE
Server returns mod error if autocomplete has nothing to complete

### ISSUE THREAD

**Issue by ghost** [thoralmighty/vs-candidemotions#3]
When I do anaction by itself, like `/me bonks` it prints `* Siinamon bonks` but in the server console it spits an error:

```
17:34:26 [Server Error] Unable to autocomplete/autocorrect phrase "bonks": System.NullReferenceException: Object reference not set to an instance of an object
```

This should not give an error if it can't autocorract/autocomplete. It should return nicely since it is not an error.

**Comment by thoralmighty** [comment]
Hmm, in the code it returns the original, unaltered phrase if an error occurred so that the user doesn't necessarily see it, but deliberately logs to the console as an FYI; I'll look into that exception

**Comment by thoralmighty** [comment]
Stability fixes added in 1.2.1 should cover this, closing until resurfaced
